test/functional/Java8andUp/src/org/openj9/test/java/lang/ref/Test_ReferenceQueue.java

```diff
@@ -55,7 +55,7 @@ public void run() {
 			synchronized (rq) {
 				// store in a static so it won't be gc'ed because the jit
 				// optimized it out
-				integer = new Integer(667);
+				integer = Integer.valueOf(667);
```

Comment:
Similar concerns with this test

Reply:
Single use, and realistically, no one is caching 667.